corehq/apps/aggregate_ucrs/models.py

```diff
@@ -116,7 +115,7 @@ def get_primary_column_adapters(self):
     def pk_columns(self):
         columns = []
         for col in self.get_columns():
-            if col.is_primary_key():
+            if col.is_primary_key:
```

Comment:
this was never triggered previously since the generator returned by `get_columns` was memoized and had already been consumed.